src/gui/qgspropertyoverridebutton.cpp

```diff
@@ -180,26 +178,6 @@ void QgsPropertyOverrideButton::updateFieldLists()
           break;
       }
 
-      switch ( f.type() )
```

Comment:
@nyalldawson , the removed code wasn't needed at all? I can't see a replacement for it.

Reply:
nope, it was unused anywhere